options/locale/locale_en-US.ini

```
@@ -1450,6 +1450,8 @@ branch.restore_failed = Failed to restore branch '%s'.
 branch.protected_deletion_failed = Branch '%s' is protected. It cannot be deleted.
 branch.restore = Restore Branch '%s'
 branch.download = Download Branch '%s'
+branch.included_desc = The default branch contains this branch
```

```suggestion
branch.included_desc = This branch is part of the default branch
```
May be this is better. Or `This branch is included in the default branch`.